.gitignore

```diff
@@ -29,7 +29,6 @@ yalc.lock
 
 contractsInfo.json
 .DS_Store
-yarn-error.log
```

Comment:
I would keep this for now just in case someone accidentally runs yarn out of habit. Won't hurt to keep it here I think